test_package/src/AbstractProcessorTest.cpp

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
I realized I forgot to add this conan test_package/ folder that comes with an example **AbstractProcessorTest.cpp**, which is the same one from the MiNiFi GTESTs. This helps us test if features are available from the MiNiFi conan package. We can simulate if we were to add MiNiFi C++ library to a C++ project, would the features we need be available to it. For now, I just copied this AbstractProcessorTest.cpp GTEST example. This test_package/ comes with CMakeLists.txt and its own conanfile.py too.

Reply:
I think we should run the same unit tests as we would run normally, even in a Conan build. I'm not a fan of duplicating a single test and introducing a new test package for conan builds, it adds too much maintenance burden.

Reply:
We can update the conanfile.py for the test_package to try to run existing MiNiFi C++ GTESTs, so we dont duplicate the already created tests. I was mainly copying over a MiNiFi test here for a quick check. I will try **cmake.ctest()**, so I can get rid of this copied test program.

Reply:
I decided to leave out the **cmake.ctest()** from being run automatically after conan finishes building MiNiFi C++, so the user will still need to go into **build_conan** folder and then run **ctest**. For the conan build and conan create approaches for MiNiFi C++, **217/219** TESTs PASS. For standalone CMake approach, **275/275** TESTs PASS.